static/app/views/releases/list/releasesPromo.tsx

```diff
@@ -94,28 +95,24 @@ type Props = {
   project: Project;
 };
 
-const ReleasesPromo = ({organization, project}: Props) => {
-  const {data, renderComponent, isLoading} = useApiRequests<{
-    internalIntegrations: SentryApp[];
-  }>({
-    endpoints: [
-      [
-        'internalIntegrations',
-        `/organizations/${organization.slug}/sentry-apps/`,
-        {query: {status: 'internal'}},
-      ],
-    ],
-  });
+function ReleasesPromo({organization, project}: Props) {
+  const {data, isLoading} = useApiQuery<SentryApp[]>(
+    [`/organizations/${organization.slug}/sentry-apps/`, {query: {status: 'internal'}}],
+    {
+      staleTime: 0,
```

Comment:
we probably need cacheTime: 0 too since local state is used that could do pop-in the next time the cache is used